python/cudf/cudf/_libxx/lib.pxd

```diff
@@ -5,26 +5,22 @@
 # cython: embedsignature = True
 # cython: language_level = 3
 
-from libc.stdint cimport int32_t, uint32_t
+import numpy as np
+
+from libc.stdint cimport *
 from libcpp cimport bool
 from libcpp.vector cimport vector
 from libcpp.memory cimport unique_ptr
+from libcpp.string cimport string
 from libcpp.pair cimport pair
 
 from rmm._lib.device_buffer cimport device_buffer, DeviceBuffer, move
 
+
 cdef extern from "cudf/types.hpp" namespace "cudf" nogil:
     ctypedef int32_t size_type
     ctypedef uint32_t bitmask_type
 
-    ctypedef enum order:
-        ASCENDING "cudf::order::ASCENDING"
```

Comment:
Where are these?

Reply:
They are defined below - I suspect it might be my fault that they were defined twice (a merge gone wrong)

Reply:
I thought they were moved, thank you.